custom-tests.yaml

```
@@ -1523,6 +1523,8 @@ api:
       };
 
       var instance = new constructor({iceServers: []});
+  RTCSessionDescription:
+    __base: 'var instance = new RTCSessionDescription({type: offer});'
```

This didn't work because of missing quotes around "offer":

```suggestion
    __base: var instance = new RTCSessionDescription({type: 'offer'});
```